package.json

```diff
@@ -10,9 +10,15 @@
     "export": "next build && next export"
   },
   "dependencies": {
+    "@erc725/erc725.js": "^0.14.4",
+    "@lukso/lsp-smart-contracts": "^0.6.2",
+    "@types/web3": "^1.2.2",
+    "ethers": "^5.6.9",
     "next": "12.2.4",
     "react": "18.2.0",
-    "react-dom": "18.2.0"
+    "react-dom": "18.2.0",
+    "react-icons": "^4.4.0",
+    "web3": "^1.7.5"
```

Comment:
So we use both `web3` and `ethers` ?

Reply:
Yes, I like ethers better specially with typescript because web3.js is not very typescript friendly so I started working with ethers but ran into some issues with the ethers provider and erc725.
Thought we might fix the erc725.js problem so I can fully switch to ethers but if you prefer I can do the entire project with web3.js?

Reply:
Yeah makes sense with Ethers - let's merge to only Ethers once erc725js is fixed